index.html

```diff
@@ -1,18 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf8" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Coffee Break Quiz</title>
+    <link rel="stylesheet" href="./public/style.css" />
     <link rel="icon" href="./public/favicon.png" type="image/png" />
-    <link href="./public/style.css" rel="stylesheet" />
   </head>
-
   <body>
-    <!-- Put your HTML here -->
-    <div id="user-interface" class="centered"></div>
```

Comment:
Removing this `div` is ill advice. It is used to show dynamic elements in the UI.